From 2c864807ade75d6ffb6d7162e89f159d2c826f51 Mon Sep 17 00:00:00 2001
From: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
Date: Mon, 22 Jun 2020 19:23:25 +0530
Subject: [PATCH 1393/1851] spi: spi-cadence: Fix mixed_enum_type coverity
 warning

commit cf5176720cefd08b874473010a7ffa7a8fac5481 from
https://github.com/Xilinx/linux-xlnx.git

Fix "Event mixed_enum_type: enumerated type mixed with another type"
coverity warning.

Signed-off-by: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/spi/spi-cadence.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/spi/spi-cadence.c b/drivers/spi/spi-cadence.c
index ff9fa4c27f62..19ad0fb81a2a 100644
--- a/drivers/spi/spi-cadence.c
+++ b/drivers/spi/spi-cadence.c
@@ -341,7 +341,8 @@ static irqreturn_t cdns_spi_irq(int irq, void *dev_id)
 {
 	struct spi_master *master = dev_id;
 	struct cdns_spi *xspi = spi_master_get_devdata(master);
-	u32 intr_status, status;
+	u32 intr_status;
+	irqreturn_t status;
 
 	status = IRQ_NONE;
 	intr_status = cdns_spi_read(xspi, CDNS_SPI_ISR);
-- 
2.31.1

